aten/src/ATen/native/Distance.cpp

```diff
@@ -5,7 +5,6 @@
 namespace at { namespace native {
 
 Tensor pairwise_distance(const Tensor& x1, const Tensor& x2, double p, double eps) {
```

Comment:
nit: Can we have this (and the public API for pairwise_distance) take in a `keepdim` option like the rest of our reduction functions, so that if we do break any user's code, it'll be a one-keyword fix?

Reply:
Isn't it already a super simple fix? (squeeze) Is this worth it?

Reply:
I think it's worth it to be consistent with the rest of our reduction API. I'm not sure what people use pairwise_distance for in real life though.